template/Dockerfile

```diff
@@ -32,7 +33,7 @@ RUN \
     # Build an empty platformio project to force platformio to install all fw build dependencies
     # The return-code will be non-zero since there's nothing to build.
     && (platformio run -d /opt/pio; echo "Done") \
-    && rm -rf /opt/pio/__PATCH_AARCH64__
+    && rm -rf /opt/pio/
```

Comment:
The aarch64 patch has been removed because it appears not to be necessary anymore.

At least on the RPi v3 aarch64 I tried the existing PIO toolchain worked fine.